source/includes/fact-oplog-background.rst

```diff
@@ -1,7 +1,15 @@
-The :term:`oplog` in the source cluster must be large enough to track
-events that happen during the time it takes to complete the initial
-sync to the destination cluster.
+
+``mongosync`` applies operations in the ``oplog`` on the source cluster
+to the data on the destination cluster.  When older operations 
+that ``mongosync`` has not applied roll off the ``oplog`` 
+on the source cluster, the sync fails and ``mongosync`` exits.
+
+During initial sync, ``mongosync`` may apply operations at a slower
+rate. After ``mongosync`` completes the initial sync, it applies changes 
+faster and stays more current in the source cluster ``oplog``.
 
 If you anticipate syncing a large data set, or if you plan to pause
-synchronization for an extended period of time, increase the size of the
-replica set ``oplog`` in the source cluster.
+synchronization for an extended period of time, you might exceed the
+:term:`oplog window`. Use the :setting:`~replication.oplogSizeMB` setting
+to increase the size of the ``oplog`` on the source cluster.
```

Comment:
(question)
to increase the size of the oplog, or the oplog window?

Reply:
The oplog window is the amount of time between the first and last entries in the oplog.  That time range is how long you have to read an entry from the oplog before it rolls off the end and is lost.  So, increasing the MB size of the oplog also increases the size of the window.